test/check-machines-create

```diff
@@ -1057,6 +1057,9 @@ class TestMachinesCreate(VirtualMachinesCase):
             root.find('os').find('resources').find('minimum').find('storage').text = '134217728'
             if self.machine.image not in ["debian-stable"]:
                 root.find('os').find('eol-date').text = '9999-12-31'
+            else:
+                eol_date = ET.SubElement(root.find('os'), 'eol-date')
```

Comment:
Do these two queries do something different actually?

Reply:
In most distro the fedora28 entry in osinfo-db contains the eol-date entry already. In debian-stable the entry is missing completely. That's the only difference, the end result after the modification is the same.